### PR TITLE
[fix] optimum token selector do sample defaults

### PR DESCRIPTION
**Pull request by tosterberg**
## Description ##

Fixing error with default behavior when do_sample is true but neither top_p / top_k are set. Fix added for defaulting the prefill/decode bool to None inside the NiaveRollingBatcher to support the updates to Optimum modeling. Additionally, removing stale model tests.

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: engines/python/setup/djl_python/transformers_neuronx_scheduler/optimum_neuron_scheduler.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def make_generations(
         return generation, finish_reason
 
     def _generate_token(self, inputs: GenerationInputs,
-                        prefill: Optional[bool]) -> List[Generation]:
+                        prefill: Optional[bool] = None) -> List[Generation]:
```
Comment (sindhuvahinis) [inline engines/python/setup/djl_python/transformers_neuronx_scheduler/optimum_neuron_scheduler.py:181]: Is this the error for do_sample?

Comment (tosterberg) [reply engines/python/setup/djl_python/transformers_neuronx_scheduler/optimum_neuron_scheduler.py:181]: It is the error for the unit test with the tiny models, I can move it out into a separate PR if you would like. I forgot that I had added that fix here.

Comment (sindhuvahinis) [reply engines/python/setup/djl_python/transformers_neuronx_scheduler/optimum_neuron_scheduler.py:182]: No that's okay

**Review by sindhuvahinis — APPROVED** [review]
